Java/PlatinumGenomes-variant-transformation/src/test/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentDataTest.java

```diff
@@ -63,17 +66,40 @@ public void testFilterVariantCallsFn() {
         ListValue.newBuilder()
             .addValues(Value.newBuilder().setStringValue("VQSRTrancheSNP99.90to100.00").build())
```

Comment:
Just out of curiosity, where in the API is the tranche level of a variant used?  I didn't think recalibration is being performed by the API and doesn't seem to affect this test.